addOns/encoder/src/main/javahelp/org/zaproxy/addon/encoder/resources/help/contents/encoder.html

```diff
@@ -31,6 +31,16 @@ <H2>Toolbar</H2>
   <li>Help - Will open the User Guide, with the content pane preset to the Encode/Decode/Hash add-on's help page.</li>
 </ul>
 
+<H2>Output Panel(s) Context Menu</H2>
+
+<ul>
+  <li>Find - Brings up a standard input dialog to allow searching within a text area.</li>
+  <li>Copy - Standard copy functionality.</li>
+  <li>Paste - Standard paste functionality.</li>
+  <li>Delete Output Panel - Removes the relevant output text area.</li>
+  <li>Replace Input Text - Replaces the content of the input text area with the relevant output text area's content. (Only active if the text area has content.)</li>
```

Comment:
It would be great if it took into account the selection, otherwise use the whole content.

Reply:
I've changed this to "replaceSelection" however that means changing the name of the menu item I think.
- Insert in Input Text
- Send to Input Text

I dunno. The behavior is insert at the insertion point unless there's a selection in which case it replaces.

Reply:
I've gone with "Send to Input Text" for now, but I'm by no means set on it.

Reply:
Sorry I was not clear, I was actually referring to the selection in the output text area, say you decode a base64 value and want to further process part of it.

Reply:
Hmm okay, I can change it that way too.